README_STAC_MODEL.md

```diff
@@ -47,54 +44,66 @@ stac-model --help
 stac-model
 ```
 
-This will make [this example item](./examples/item_basic.json) for an example model.
+This will make [this example item](./examples/item_basic.json) for an example
+model.
 
 ## 📈 Releases
 
-You can see the list of available releases on the [GitHub Releases][github-releases] page.
+You can see the list of available releases on the [GitHub
+Releases][github-releases] page.
 
 ## 📄 License
+
 [![License][blic1]][blic2]
 
-This project is licenced under the terms of the `Apache Software License 2.0` licence.
-See [LICENSE][blic2] for more details.
+This project is licenced under the terms of the `Apache Software License 2.0`
+licence. See [LICENSE][blic2] for more details.
 
 ## 💗 Credits
+
 [![Python project templated from galactipy.][bp6]][bp7]
 
 <!-- Anchors -->
 
 [bp1]: https://img.shields.io/pypi/pyversions/stac-model?style=for-the-badge
 [bp2]: https://pypi.org/project/stac-model/
-[bp3]: https://img.shields.io/pypi/v/stac-model?style=for-the-badge&logo=pypi&color=3775a9
+[bp3]:
+  https://img.shields.io/pypi/v/stac-model?style=for-the-badge&logo=pypi&color=3775a9
```

Comment:
these newlines seems to render correctly, but the diff doesn't highlight them properly, so not sure if this is valid/desired